algorithms/unit_tests/TestSort.hpp

```diff
@@ -425,6 +425,43 @@ void test_sort(unsigned int N) {
   test_sort_integer_overflow<ExecutionSpace, unsigned long long>();
   test_sort_integer_overflow<ExecutionSpace, int>();
 }
+
+template <class ExecutionSpace>
+void test_sort_empty_view() {
+  // does not matter if we use int or something else
+  Kokkos::View<int*, ExecutionSpace> v;
```

Comment:
```suggestion
  Kokkos::View<int*, ExecutionSpace> v("v", 0);
```

Reply:
this slipped , thought i did it too. thanks !